CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ and this project adheres to
 - Install security updates in project Docker images
 - Model selector to retrieve associated pydantic model of a given event
 - `validate` command to lint edx events using pydantic models
+- The elasticsearch backend now supports all available bulk operation types
```

Comment:
I would prefer an infinitive formulation in the Changelog. WDYT?

Reply:
Better?